splunk_add_on_ucc_framework/install_python_libraries.py

```diff
@@ -29,6 +29,7 @@
 
 
 LIBS_REQUIRED_FOR_UI = {"splunktaucclib": "6.4.0"}
+LIBS_REQUIRED_FOR_OAUTH = {"solnlib": "5.5.0"}
```

Comment:
Let's leave the OAuth code updates for the next PRs, it's quite hard to review this one and I am not totally sure we need to require a specific version of `solnlib` for updating OAuth-related handlers.

Reply:
For the OAuth helper that we are generating, the `solnlib.conf_manager`'s `get_proxy_dict` is being used, hence I've fixed the solnlib version and OAuth template file with this Proxy related PR.